.gitignore

```diff
@@ -105,4 +105,11 @@ hs_err_pid*
 .vscode/
 
 # Python Virtual Env
-.venv
\ No newline at end of file
+.venv
+
+# Development Experience
+
+# You can create runtime/defaults/src/main/resources/application-dev.properties file
+# to override default properties for local development.
+# And then use `./gradlew run -Dquarkus.profile=dev` to run Polaris with dev profile.
+application-dev.properties
```

Comment:
`dev` is a pretty common / standard profile name in Quarkus. Ignoring it in git might clash with future use cases if/when it is actually used.

For local testing, I suppose it could be named `application-local.properties` (`-Dquarkus.profile=local`) (following helm patterns). Would that work for you, @binarycat0 ?